src/components/NavBar/index.js

```diff
@@ -39,13 +41,37 @@ class NavBar extends React.PureComponent {
               className={classes.navBarLogo}
             />
           </Link>
-          <NavEventsButtons />
+          <Link to="/" className={classes.navBarLink}>
+            <Button
+              data-index={EventStatus.Bet}
+              className={classNames(
+                classes.navEventsButton,
+                match.path === '/' ? 'selected' : '',
+              )}
+            >
+              QTUM Prediction
+            </Button>
+            { this.renderCurrentTabArrow('/') }
```

Comment:
what do you think about building a button component that has the built in arrow?
https://material-ui-next.com/demos/buttons/#complex-buttons